api/src/opentrons/protocol_api/labware.py

```diff
@@ -785,10 +793,26 @@ def previous_tip(self, num_tips: int = 1) -> Optional[Well]:
                          column
         :type num_tips: int
         :return: The :py:class:`.Well` meeting the target criteria, or ``None``
+
+        .. deprecated:: 2.14
+            Modification of tip tracking state outside :py:meth:`.reset` has been deprecated.
```

Comment:
For consistency with https://github.com/Opentrons/opentrons/pull/12114#discussion_r1099208466, let's use `versionchanged` for hard removals.

```suggestion

        .. versionchanged:: 2.14
            This method has been removed.
```

I'd also not say "modification of tip tracking state" in this particular case, because this is just a getter and was never modifying anything to begin with. Unless you think this getter tends to be linked to modifying tip state in practice, in user protocols.